tools/data-handler/test/command-handler.test.ts

```diff
@@ -1186,8 +1183,7 @@ describe('create command', () => {
   });
   it('template with "123"', async () => {
     const templateName = 'loc/123';
-    const templateContent =
-      '{"buttonLabel": "Button1", "namePrompt": "Prompt1"}';
+    const templateContent = '{}';
```

Comment:
As `templateContent` is optional, there could be a test that doesn't pass the `content` parameter at all.
Either a new test, or encapsulate to one of the existing ones.

Reply:
Actually, if the `content` file is always empty, shouldn't we remove the whole parameter from CLI?

Reply:
I'll add a new test